playpen.py

```diff
@@ -1,21 +1,40 @@
 #!/usr/bin/env python3
 
 import subprocess
+import re, os.path
 
-def execute(version, command, arguments, data=None):
-    with subprocess.Popen(("playpen",
-                           "root-" + version,
+g_wrapper_cmd_cache = {}
+
+def execute(channel, command, arguments, data=None, skip_playpen=False, debug_me=False):
+    cache_key = (channel, command)
+    wrapper_cmd = g_wrapper_cmd_cache.get(cache_key, None)
+    if wrapper_cmd is None:
+        if skip_playpen:
+            wrapper_cmd = ()
+            if os.path.exists(command):
+                with open(command) as fp:
+                    if re.match(r'#!.*sh', fp.readline()):
+                        wrapper_cmd = ("sh", )  # compile.sh has non-standard #!/usr/bin/dash
```

Comment:
Arch symlinks `/bin` to `/usr/bin` and `/sbin` to `/usr/sbin`, so that might be what you're seeing. You should be able to use `/bin/sh`, since its included in bash's package.